app/models/github_user.rb

```diff
@@ -10,27 +11,9 @@ def initialize(attrs = {})
     @starred_url = attrs[:starred_url]
   end
 
-  # def self.find_all_by(filter = {})
-  # SunlightService.legislators_by(filter).map do |raw_legislator|
-  #   new(raw_legislator)
-  # end
-
-   def new_user(access_token)
-     GithubService.get_user(access_token).map do |raw_user|
-       new(raw_user)
-   end
-end
+  def self.create_github_user(access_token)
```

Comment:
since we're within the `GithubUser` class, i'd call this method `.create` - that way we aren't repeating ourselves when calling the method